<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animasyon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblGoruntu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>528</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOynat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Oynat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDurdur">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Durdur</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
